addons/web/static/src/scss/bootstrap_review.scss

```diff
@@ -324,3 +324,10 @@ $-color-for-gray-200-bg: adjust-color-to-background($body-color, $gray-200);
         }
     }
 }
+
+// Button within input-group (e.g., "search bar")
+.input-group {
+    .btn:first-child, .btn:last-child {
```

Comment:
Just a question (as I was not present when the specs were discussed): do we want that only for buttons or for anything inside an input-group ? Because when looking in the `bootsrap.css` file (for the record, I was looking for the rule that sets the radius only on the external corners, so the one that sets the inside ones to 0) we see that the rules are more general than just for the buttons (first and last child in general) :arrow_down: So maybe it should be something similar here ? :thinking: :slightly_smiling_face: 
```css
.input-group:not(.has-validation) > :not(:last-child):not(.dropdown-toggle):not(.dropdown-menu),
.input-group:not(.has-validation) > .dropdown-toggle:nth-last-child(n+3) {
  border-top-right-radius: 0;
  border-bottom-right-radius: 0;
}
.input-group.has-validation > :nth-last-child(n+3):not(.dropdown-toggle):not(.dropdown-menu),
.input-group.has-validation > .dropdown-toggle:nth-last-child(n+4) {
  border-top-right-radius: 0;
  border-bottom-right-radius: 0;
}
.input-group > :not(:first-child):not(.dropdown-menu):not(.valid-tooltip):not(.valid-feedback):not(.invalid-tooltip):not(.invalid-feedback) {
  margin-left: -1px;
  border-top-left-radius: 0;
  border-bottom-left-radius: 0;
}
```

Reply:
The rules you show here are used to cancel border radius for elements that are not the first or last within `input-group`. It works well for buttons, see here:

![image](https://github.com/odoo/odoo/assets/52911687/a6f17f29-2d4e-46b2-8ef8-d59fec5ba51f)

Reply:
Yes what I meant is that these rules do not target buttons specifically, it just targets the children of an input-group and filters what should not be considered (like the dropdowns)
So what I was asking is if the rule you added should also consider any child of an input-group or if we just want it for the buttons :slightly_smiling_face: 

But as discussed IRL, we should just focus on the issue to fix (so the search bar) since we are in stable, so it is okay for now to only target the buttons I guess :+1: